### PR TITLE
Details/Summary: Optimized indentation rules

### PR DESCRIPTION
**Pull request by LaurentGoderre**
Fixes the issue raised by #6530

### CODE REVIEW

**Comment by EricDunsworth** [comment]
FYI I noticed that with this PR's changes, when I disable JS in Firefox, the summaries get cut off on the left due to the negative margins. Some extra changes might need to be added in _noscript.scss to prevent it.

File: src/plugins/tabs/_noscript.scss [hunk 1]
```diff
@@ -6,11 +6,12 @@
 /**
  * Mobile-friendly styles
  */
- 
+
 %tabs-noscript-details-summary-display-block-important {
 	> {
 		details {
 			display: block;
+			padding-left: $details-identation;
```
Comment (LaurentGoderre) [inline src/plugins/tabs/_noscript.scss]: There you go @EricDunsworth

**Comment by LaurentGoderre** [comment]
@EricDunsworth can you test now?

**Comment by EricDunsworth** [comment]
@LaurentGoderre Gonna check it out in a sec.

File: src/plugins/tabs/_noscript.scss (hunk 1 above)
Comment (EricDunsworth) [inline src/plugins/tabs/_noscript.scss]: It looks like the left padding is around 2x wider in small view now (compared to what's currently on github.io). In medium and bigger views the summaries look like they're going right up against the border.

Maybe there'd be value in removing that line and adding something like ``summary {margin-left: 0;}`` that only targets medium view and larger when JS is disabled.

**Comment by LaurentGoderre** [comment]
@EricDunsworth any update?

**Comment by EricDunsworth** [comment]
@LaurentGoderre I ended up directly replying to your note above on Friday. Scroll up a bit to see it :).

File: src/plugins/tabs/_noscript.scss (hunk 1 above)
Comment (LaurentGoderre) [inline src/plugins/tabs/_noscript.scss]: Not sure that would fix it.....

Comment (LaurentGoderre) [inline src/plugins/tabs/_noscript.scss]: Also, the left padding is consistent across views.

**Comment by LaurentGoderre** [comment]
@EricDunsworth there you go! Your proposed change fixed it. Can you check again?

**Comment by EricDunsworth** [comment]
@LaurentGoderre Ok, gonna check it in a sec.

**Comment by EricDunsworth** [comment]
@LaurentGoderre The summary itself looks perfectly positioned in medium view+ now, but I noticed that it's still looking a bit odd in small view (with JS disabled) due to some padding from the details element appearing to the left of the summary.

Also, if there's any appetite to tackle it as a part of this PR, I should note that when JS is disabled, the content that follows summaries isn't indented. If you don't want to deal with it right now, I could send in a separate PR later to tackle that part.

**Comment by LaurentGoderre** [comment]
@EricDunsworth I am not seeing it. Can you add a screenshot?

**Comment by EricDunsworth** [comment]
@LaurentGoderre Here's a few before and after shots of http://wet-boew.github.io/v4.0-ci/demos/tabs/tabs-en.html (FF, small view with JS disabled)...

**Before:**
tabs_small-view_before

**After:**
tabs_small-view_after

**Comment by LaurentGoderre** [comment]
I only get that on hover  though no?

**Comment by LaurentGoderre** [comment]
OOOh nevermind. So many combinations!

**Comment by EricDunsworth** [comment]
Yeah lol. It's pretty crazy how many different kinds of layouts/combinations need to be tested out when tweaking details/summary heh.

**Comment by LaurentGoderre** [comment]
Ok! the tabbed interface has some weird CSS in viewports. The latest I think fixes everything *fingers crossed*

**Comment by EricDunsworth** [comment]
@LaurentGoderre Ok, gonna check it out again in a bit.

**Comment by EricDunsworth** [comment]
Looks good now IMO :)!

Once this gets merged I'll send over that "extra" indent PR when I have a chance.

**Comment by LaurentGoderre** [comment]
Ok, let's merge then! Thanks for your help!

**Comment by EricDunsworth** [comment]
And thanks for bearing with my nitpickyness lol.

**Pull request merged**